.sh-extension

```diff
@@ -1,9 +1,13 @@
 {
     "requiredFiles": [],
-    "buildLogs": [],
-    "appLogs": [
-        "appsody"
-    ],
+    "appWorkspaceLogs": {
+        "files": {
+            "appsody": null
+        }
+    },
+    "appContainerLogs": {},
+    "buildWorkspaceLogs": {},
+    "buildContainerLogs": {},
```

Comment:
I thought these are optional and can be left out if they are empty?

Reply:
It can be left out. However, I thought you wanted them there for clarity if in the future anyone wanted to add a new log file.

Reply:
ok sure, just wanted to confirm